### PR TITLE
add tags

### PR DESCRIPTION
**Pull request by intrepidcanadian**
## Pre-flight checklist

- [x ] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [ x] A relating issue is created: # (<- fill the issue number here)
- [x ] **This is NOT a PR to submit translation**

### CODE REVIEW

**Comment by darwintree** [comment]
[12:35:42.815] Running build in Washington, D.C., USA (East) – iad1
[12:35:42.918] Cloning github.com/Conflux-Chain/conflux-documentation (Branch: main, Commit: 52b0288)
[12:35:51.738] Cloning completed: 8.820s
[12:35:51.740] Running "bash scripts/checkGitCommitBash.sh"
[12:35:51.751] Warning: Due to "engines": { "node": ">=18.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("18.x") will not apply, Node.js Version "20.x" will be used instead. Learn More: http://vercel.link/node-version
[12:35:51.752] Warning: Detected "engines": { "node": ">=18.0" } in your `package.json` that will automatically upgrade when a new major Node.js Version is released. Learn More: http://vercel.link/node-version
[12:35:51.762] current branch is not l10n_main (build needed)
[12:35:56.823] Restored build cache from previous deployment (4mmB59FPV8RZNi5f2sxD3maDBKfY)
[12:35:56.959] Running "vercel build"
[12:35:58.192] Vercel CLI 37.8.0
[12:36:00.310] Warning: Due to "engines": { "node": ">=18.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("18.x") will not apply, Node.js Version "20.x" will be used instead. Learn More: http://vercel.link/node-version
[12:36:00.311] Warning: Detected "engines": { "node": ">=18.0" } in your `package.json` that will automatically upgrade when a new major Node.js Version is released. Learn More: http://vercel.link/node-version
[12:36:00.315] Installing dependencies...
[12:36:00.643] yarn install v1.22.19
[12:36:00.816] [1/5] Validating package.json...
[12:36:00.816] [2/5] Resolving packages...
[12:36:01.455] success Already up-to-date.
[12:36:01.459] Done in 0.82s.
[12:36:01.489] Running "yarn run build"
[12:36:01.798] yarn run v1.22.19
[12:36:01.831] $ yarn front-matter && docusaurus build
[12:36:02.066] $ node scripts/add-front-matter.js
[12:36:02.165] ./docs/general
[12:36:02.225] ./docs/core
[12:36:02.250] ./docs/espace
[12:36:02.274] 
[12:36:02.275] /vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:173
[12:36:02.275]   throw generateError(state, message);
[12:36:02.275]   ^
[12:36:02.277] YAMLException: duplicated mapping key at line 39, column 1:
[12:36:02.278]     displayed_sidebar: eSpaceSidebar
[12:36:02.278]     ^
[12:36:02.278]     at generateError (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:167:10)
[12:36:02.278]     at throwError (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:173:9)
[12:36:02.278]     at storeMappingPair (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:335:7)
[12:36:02.278]     at readBlockMapping (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1098:9)
[12:36:02.278]     at composeNode (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
[12:36:02.279]     at readDocument (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1525:3)
[12:36:02.279]     at loadDocuments (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1588:5)
[12:36:02.279]     at load (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1614:19)
[12:36:02.279]     at Object.safeLoad (/vercel/path0/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1637:10)
[12:36:02.279]     at module.exports (/vercel/path0/node_modules/gray-matter/lib/parse.js:12:17) {
[12:36:02.280]   reason: 'duplicated mapping key',
[12:36:02.280]   mark: Mark {
[12:36:02.280]     name: null,
[12:36:02.280]     buffer: '\n' +
[12:36:02.280]       'sidebar_position: 5\n' +
[12:36:02.287]       'title: Network RPC Endpoints\n' +
[12:36:02.289]       'description: Conflux eSpace RPC endpoints\n' +
[12:36:02.290]       'keywords:\n' +
[12:36:02.290]       '  - network endpoints\n' +
[12:36:02.290]       '  - Conflux eSpace\n' +
[12:36:02.290]       '  - RPC\n' +
[12:36:02.290]       '  - JSON-RPC\n' +
[12:36:02.291]       '  - Mainnet\n' +
[12:36:02.291]       '  - Testnet\n' +
[12:36:02.291]       '  - Chain ID\n' +
[12:36:02.291]       '  - HTTP endpoints\n' +
[12:36:02.291]       '  - WebSocket endpoints\n' +
[12:36:02.291]       '  - Confura\n' +
[12:36:02.292]       '  - Rate limits\n' +
[12:36:02.292]       '  - API key\n' +
[12:36:02.292]       '  - Web3 Paywall\n' +
[12:36:02.292]       '  - Commercial RPC service\n' +
[12:36:02.292]       '  - Blockchain explorer\n' +
[12:36:02.293]       '  - EVM compatibility\n' +
[12:36:02.293]       '  - Web3 development\n' +
[12:36:02.293]       '  - dApp infrastructure\n' +
[12:36:02.293]       '  - Blockchain node\n' +
[12:36:02.293]       '  - Public RPC service\n' +
[12:36:02.293]       '  - Enterprise solutions\n' +
[12:36:02.294]       'displayed_sidebar: eSpaceSidebar\n' +
[12:36:02.294]       'tags:\n' +
[12:36:02.294]       '  - Conflux eSpace\n' +
[12:36:02.294]       '  - RPC Endpoints\n' +
[12:36:02.294]       '  - Confura\n' +
[12:36:02.294]       '  - Mainnet\n' +
[12:36:02.297]       '  - Testnet\n' +
[12:36:02.297]       '  - API\n' +
[12:36:02.298]       '  - Web3 Infrastructure\n' +
[12:36:02.298]       '  - Blockchain Development\n' +
[12:36:02.298]       '  - Rate Limits\n' +
[12:36:02.298]       '  - Network Configuration\n' +
[12:36:02.298]       'displayed_sidebar: eSpaceSidebar\n' +
[12:36:02.298]       '\x00',
[12:36:02.298]     position: 700,
[12:36:02.299]     line: 38,
[12:36:02.299]     column: 0
[12:36:02.299]   }
[12:36:02.299] }
[12:36:02.299] 
[12:36:02.299] Node.js v20.17.0
[12:36:02.312] error Command failed with exit code 1.
[12:36:02.312] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[12:36:02.443] error Command failed with exit code 1.
[12:36:02.443] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[12:36:02.574] Error: Command "yarn run build" exited with 1
[12:36:03.101]

**Comment by darwintree** [comment]
duplicate displayed_sidebar is setting on certain article

**Comment by intrepidcanadian** [comment]
> duplicate displayed_sidebar is setting on certain article

fixed

File: docs/general/conflux-basics/ecosystem.md [hunk 1]
```diff
@@ -2,8 +2,27 @@
...
+- lending
+- borrowing
+- Meson
+- stablecoins
+- cross-chain bridge
+- Zero Gravity
+- ConfluxHub
+- CFX staking
+- governance
+-PoS validators
```
Comment (darwintree) [inline docs/general/conflux-basics/ecosystem.md:22]:
```suggestion
- PoS validators
```

**Comment by darwintree** [comment]
tags are used to filter articles and keywords for SEO optimization
image
I think the tags should be selected carefully so readers can get relating information quickly

**Comment by darwintree** [comment]
After clicking a tag, relating article will be shown. If one tag is only for one topic, it should be considered if it is meaningful. (However keywords is of different usage as readers will not see them but search engines can)

**Comment by intrepidcanadian** [comment]
thanks for the clarification on this. let me refine the tags and keywords

> After clicking a tag, relating article will be shown. If one tag is only for one topic, it should be considered if it is meaningful. (However keywords is of different usage as readers will not see them but search engines can)

**Comment by intrepidcanadian** [comment]
i have changed the tags and keywords accordingly. 

> After clicking a tag, relating article will be shown. If one tag is only for one topic, it should be considered if it is meaningful. (However keywords is of different usage as readers will not see them but search engines can)

**Review by 0xn1c0 — APPROVED** [review]

**Review by 0xn1c0 — APPROVED** [review]

**Pull request merged**